orchestration/src/main/java/com/sap/ai/sdk/orchestration/SystemMessage.java

```diff
@@ -13,5 +16,58 @@ public class SystemMessage implements Message {
   @Nonnull String role = "system";
 
   /** The content of the message. */
-  @Nonnull String content;
+  @Nonnull MessageContent content;
+
+  /**
+   * Creates a new system message from one or more strings.
+   *
+   * @param message the first message.
+   * @param additionalMessages the additional messages.
+   */
+  public SystemMessage(
+      @Nonnull final String message, @Nullable final String... additionalMessages) {
+    content = MessageContent.text(message, additionalMessages);
+  }
+
+  /**
+   * Creates a new system message with the given message content.
+   *
+   * @param messageContent the message content.
+   */
+  public SystemMessage(@Nonnull final MessageContent messageContent) {
+    content = messageContent;
+  }
+
+  /**
+   * Add text to the message.
+   *
+   * @param messages the text to add.
+   * @return the new message.
+   */
+  @Nonnull
+  public SystemMessage addText(@Nonnull final String... messages) {
```

Comment:
while immutability can be nice, the verb `add` kinda suggests a mutating behavior.

e.g. if I run:

```java
var msg = Message.system("Foo");
msg.addText(" and Bar");
System.out.println(msg);
```

this printing just `Foo` feels unexpected. Maybe we find a better naming, e.g. `andText` which sounds more like a builder..